examples/onlineboutiquemicroservice/README.md

```diff
@@ -0,0 +1,59 @@
+| [loadgenerator](https://github.com/GoogleCloudPlatform/microservices-demo/tree/master/src/loadgenerator)                 | Python/Locust | Continuously sends requests imitating realistic user shopping flows to the frontend.                                              |
+
+## Run the Example
+
+### Prerequisites
+#### Docker and Kubernetes
+- Use [Docker for Mac](https://docs.docker.com/docker-for-mac/install/) to install on Mac.
+- Use [Docker for Windows](https://docs.docker.com/docker-for-windows/) to install on Windows.
+- Install [Docker CE](https://docs.docker.com/v17.12/install/#server) and [Minikube](https://kubernetes.io/docs/tasks/tools/install-minikube/) on Linux.
+
+**Note**: Execute `eval $(minikube docker-env)` after installing Minikube so
+that the Docker client points to Minikube’s docker registry.
+
+For this example to work, all the 10 microservices must be running. Therefore, you have to run the following services from [cloud-native microservices demo application](https://github.com/GoogleCloudPlatform/microservices-demo).
+* [Frontend](https://github.com/GoogleCloudPlatform/microservices-demo/tree/master/src/frontend)
+* [Cart Service](https://github.com/GoogleCloudPlatform/microservices-demo/tree/master/src/cartservice)
+* [Currency Service](https://github.com/GoogleCloudPlatform/microservices-demo/tree/master/src/currencyservice)
+* [Payment Service](https://github.com/GoogleCloudPlatform/microservices-demo/tree/master/src/paymentservice)
+* [Shipping Service](https://github.com/GoogleCloudPlatform/microservices-demo/tree/master/src/shippingservice)
+* [Email Service](https://github.com/GoogleCloudPlatform/microservices-demo/tree/master/src/emailservice)
+* [Checkout Service](https://github.com/GoogleCloudPlatform/microservices-demo/tree/master/src/checkoutservice)
+* [Loadgenerator](https://github.com/GoogleCloudPlatform/microservices-demo/tree/master/src/loadgenerator)
+
+1. Clone the [cloud-native microservices demo application](https://github.com/GoogleCloudPlatform/microservices-demo) of [Google Cloud Platform](https://github.com/GoogleCloudPlatform) and continue to the repository directory.
+2. The `kubernetes-manifests` file contains the deploying instructions for all the 10 services. Therefore, first remove the content related to the `adservice, recommendationservice` and `productcatalogservice`.
+3. Run `kubectl apply -f ./release/kubernetes-manifests.yaml`to deploy the 7 services.
+   
+    Note: You can run `kubectl delete -f ./release/kubernetes-manifests.yaml` to stop the 7 services when in need.
+4. Now clone this repository and run `sh examples/onlineboutiquemicroservice/scripts/start.sh`(This will build the 3 services and deploy in kubernetes).
+5. Run `kubectl get pods` to see whether pods are in a Ready state.
+6. If all the pods are running, run `kubectl get service/frontend-external`to get the frontend-external ip.
```

Comment:
```suggestion
6. If all the pods are running, run `kubectl get service/frontend-external` to get the frontend-external ip.
```

Reply:
Changed in https://github.com/ballerina-platform/module-ballerina-grpc/pull/392/commits/56b949cc1bb433c00a38ac181b0b233f12087e9b